test/it/group-get.js

```diff
@@ -2,9 +2,11 @@ const utils = require('../utils');
 const okta = require('../../');
 const expect = require('chai').expect;
 let orgUrl = process.env.OKTA_CLIENT_ORGURL;
+let mockServer = false;
 
 if (process.env.OKTA_USE_MOCK) {
   orgUrl = `${orgUrl}/group-get-and-stats`;
+  mockServer = true;
 }
```

Comment:
We may want to pull this out into a util method or parent class, it looks like it is reused in each test